tests/test_blas_ops.py

```diff
@@ -404,3 +404,146 @@ def test_accuracy_addr(M, N, dtype):
         res_out = torch.addr(input_tensor, vec1, vec2, alpha=alpha, beta=beta)
 
     gems_assert_close(res_out, ref_out, dtype, equal_nan=True)
+
+
+def get_test_params():
+    import itertools
+
+    MNK_FACTORS = [
+        (1, 256, 128),
+        (16, 256, 496),
```

Comment:
did you mean `4096` when you type `496`?
I'm asking because all other numbers in this constant are power of 2.